khl/requester.py

```diff
@@ -15,9 +15,11 @@
 class HTTPRequester:
     """wrap raw requests, handle boilerplate param filling works"""
 
-    def __init__(self, cert: Cert):
+    def __init__(self, cert: Cert, ratelimit: bool = True):
         self._cert = cert
         self._cs: Union[ClientSession, None] = None
+        self._ratelimit = self.RateLimit()
```

Comment:
naming:

作为一个组件，用名词更好
```suggestion
        self._ratelimiter = self.RateLimiter()
```